<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>requestItem</class>
 <widget class="QWidget" name="requestItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>49</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="userLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="amountLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="timeLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="acceptBtn">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #598bb8;   /* dark blue */
    color: white;
    border: none;                /* no border line */
    border-radius: 12px;         /* rounded corners */
    padding: 8px 16px;
    font-size: 11px;
    background-image: none;      /* no image */
	max-width: 100px;

}

QPushButton:hover {
    background-color: #0D1B5A;   /* darker on hover */
}</string>
     </property>
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="rejectBtn">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #598bb8;   /* dark blue */
    color: white;
    border: none;                /* no border line */
    border-radius: 12px;         /* rounded corners */
    padding: 8px 16px;
    font-size: 11px;
    background-image: none;      /* no image */
	max-width: 100px;

}

QPushButton:hover {
    background-color: #0D1B5A;   /* darker on hover */
}</string>
     </property>
     <property name="text">
      <string>Reject</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
